lib/plugins/create/templates/aws-go/gitignore

```diff
@@ -0,0 +1,4 @@
+bin
```

Comment:
IMO we should document this folder as well.
Suggestion: `# golang output binary directory`